Assets/HoloToolkit/Utilities/Scripts/RaycastHelper.cs

```diff
@@ -0,0 +1,312 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public static class RaycastHelper
```

Comment:
I think we already have a similar class for Raycasting Extensions.

Reply:
I thought we were good on this one. The only Raycast class I can find in MRTK is for comparisons. *RaycastResultComparer.cs*

Reply:
@StephenHodgson I think it might save everyone time if we adopt a policy where blocking change requests based on design principles or development philosophy must be accompanied by a suggested alternative.

Also keep in mind that this is being merged into the dev branch, which was created to support more rapid / flexible merging. We can always revisit this issue before merging to master.

Reply:
Some of the CRs I suggested in this pass were for valid reasons (i.e. the missing UNITY_WSA compiler definitions for WSA namespace usages.

Reply:
Right, those don't seem to be a problem.